public/index.html

```diff
@@ -223,5 +262,198 @@ <h2>Resources</h2>
+        <pattern
+          id="pattern-few-protections"
+          x="0"
+          y="0"
+          width="8"
+          height="8"
+          patternUnits="userSpaceOnUse"
+          patternContentUnits="userSpaceOnUse"
+          patternTransform="rotate(45)"
+        >
+          <path
+            stroke="#d9f0a3"
+            stroke-opacity="1"
+            stroke-width="4"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            fill="none"
+            pointer-events="none"
+            d="M0 2 H 8"
+          ></path>
+          <path
+            stroke="#ffffff"
+            stroke-opacity="0"
+            stroke-width="4"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            fill="none"
+            pointer-events="none"
+            d="M0 6 H 8"
+          ></path>
+        </pattern>
+        <pattern
+          id="pattern-some-protections"
+          x="0"
+          y="0"
+          width="8"
+          height="8"
+          patternUnits="userSpaceOnUse"
+          patternContentUnits="userSpaceOnUse"
+          patternTransform="rotate(45)"
+        >
+          <path
+            stroke="#78c679"
+            stroke-opacity="1"
+            stroke-width="4"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            fill="none"
+            pointer-events="none"
+            d="M0 2 H 8"
+          ></path>
+          <path
+            stroke="#ffffff"
+            stroke-opacity="0"
+            stroke-width="4"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            fill="none"
+            pointer-events="none"
+            d="M0 6 H 8"
+          ></path>
+        </pattern>
+        <pattern
+          id="pattern-many-protections"
+          x="0"
+          y="0"
+          width="8"
+          height="8"
+          patternUnits="userSpaceOnUse"
+          patternContentUnits="userSpaceOnUse"
+          patternTransform="rotate(45)"
+        >
+          <path
+            stroke="#238443"
+            stroke-opacity="1"
+            stroke-width="4"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            fill="none"
+            pointer-events="none"
+            d="M0 2 H 8"
+          ></path>
+          <path
+            stroke="#ffffff"
+            stroke-opacity="0"
+            stroke-width="4"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            fill="none"
+            pointer-events="none"
+            d="M0 6 H 8"
+          ></path>
+        </pattern>
+
+        <!-- city protections have a slightly modified pattern to better differentiate them -->
+        <pattern
+          id="pattern-few-protections--city-level"
+          x="0"
+          y="0"
+          width="8"
+          height="8"
+          patternUnits="userSpaceOnUse"
+          patternContentUnits="userSpaceOnUse"
+          patternTransform="rotate(135)"
```

Comment:
Seems like a good fix and works visually imo.

Reply:
@timhitchins would you mind saying more about this?